base/docs/basedocs.jl

```diff
@@ -447,6 +447,23 @@ julia> A'
 """
 kw"'"
 
+"""
+    "\$"
+
+The string interpolation operator, see [string](@ref string-interpolation)
+and [expression](@ref man-expression-interpolation).
+
+# Examples
+```jldoctest
+julia> name = "Joe"
+"Joe"
+
+julia> "My name is $name."
```

Comment:
We completely forgot, there's a `$` here too. We need to escape this as well.

Reply:
You can easily check if you've done correctly or not by using the `println` function.

```julia
julia> some_str = """My name is $name."""
```
This will produce an error:
```
UndefVarError: name not defined
```

Reply:
This is the whole example.

```julia
# Examples
julia> name = "Joe"
"Joe"

julia> "My name is $name."
"My name is Joe."
```
Output

```console
julia> name = "Joe"
"Joe"

julia> """My name is $name"""
"My name is Joe"
```

Reply:
See the point is, you are writing this inside triple quotes `"""` which translates to string. Here Julia needs to process these strings and hence in that process the `$` should be escaped. Else, it will look for `name` which you don't have. I hope you understand.

Nevertheless, can you make the changes I suggested and push, if it all goes well the build should be green.

Reply:
I think the compiler allow using `$` and I now I get it. Thanks